### DISCUSSION TITLE
Intel performance libraries

### DISCUSSION THREAD

**Discussion by L-Curr0** [GeoscienceAustralia/DynAdjust#226] [Q&A] [answered]
Hi everyone,
Noting the use of Intel's performance libraries in the software licence - has there been any reported or perceived, compatibility issues or performance degradation when using AMD CPUs?
Thanks, Liam.

**Comment by nicgowans** [comment, marked as answer]
Hi Liam,

In some AWS testing I did a couple years ago, I found DynAdjust v1.2.3 performed around 2.5 x faster per iteration on a 2.3 GHz Intel Xeon E7-8880 v3 processor compared against a 2.5 GHz AMD EPYC 7000.

In terms of actual iteration times, it was ~1.35 hours on the Intel processor vs ~3.4 hours on the AMD.

Didn't run into any compatibility issues.

Nic

**Reply by L-Curr0** [reply]
Thanks Nic, certainly useful info, depending on use case etc.
